hack/tools.go

```diff
@@ -24,6 +24,9 @@ import (
 	_ "knative.dev/pkg/configmap/hash-gen"
 	_ "knative.dev/pkg/hack"
 
+	// Needed for the storage version migration.
+	_ "knative.dev/pkg/apiextensions/storageversion/cmd/migrate"
```

Comment:
:facepalm:

Reply:
I added a test for these kinds of problems: https://github.com/knative-sandbox/eventing-kafka-broker/pull/1976